doc/content/about/theme-team.toml

```diff
@@ -1,59 +1,49 @@
 [[item]]
 type = 'card'
 classcard = 'text-center'
-body = '''
-{{< image
-    src='https://avatars.githubusercontent.com/u/601365?v=4"'
-    alt="Avatar of Adam Porter"
```

Comment:
Upon regenerating, Adam (not pinging them as I'm not sure if they're involved) is no longer a part of the organisation team. Is this alright?